data-collection/deploy/deploy-data-collection.yaml

```diff
@@ -259,6 +269,7 @@ Conditions:
       - !Condition DeployBackupModule
       - !Condition DeployTransitGatewayModule
       - !Condition DeployCostOptimizationHubModule
+    - !Condition DeployAWSFeedsModule
```

Comment:
not aligned?

Reply:
I have updated the code by eliminating dependency from AccountCollector as this feature is not required for AWS Feeds module